decidim-core/lib/decidim/core/test/shared_examples/manage_moderations_examples.rb

```diff
@@ -26,7 +26,7 @@
     it "user can review them" do
       moderations.each do |moderation|
         within "tr[data-id=\"#{moderation.id}\"]" do
-          expect(page).to have_content moderation.reportable.reported_content_url
+          expect(page).to have_css("a[href='#{url}']")
```

Comment:
Tests are complaining because `url` is undefined, can you replace `url` with `moderation.reportable.reported_content_url`? :smile:

Reply:
Ups!! :sweat_smile: